setup.py

```diff
@@ -75,6 +75,7 @@
     'sympy.codegen',
     'sympy.combinatorics',
     'sympy.concrete',
+    'sympy.control',
```

Comment:
I think that this should be under `physics` or we should create a new package called `engineering`. Right now engineering related things are put under `physics`.